src/CoordRefSystems.jl

```diff
@@ -28,6 +29,10 @@ include("codes.jl")
 include("strings.jl")
 include("get.jl")
 
+function __init__()
+  _init_epsg_datadep_registration()
```

Comment:
please avoid underscore in function names, this can be renamed to `epsgregistration()` or similar name